packages/protocol/contracts-0.8/governance/Validators.sol

```diff
@@ -1497,6 +1498,17 @@ contract Validators is
     return true;
   }
 
+  function _sendValidatorPaymentIfNecessary(address validator) private {
+    getEpochManager().sendValidatorPayment(validator);
```

Comment:
should this avoid the call completely in L2?

Reply:
are we sure this actually doesn't rever on L1?

Reply:
sendValidatorPayment should probably be enabled if the epoch system is initialized